src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/RazorCompletionItemKind.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.CodeAnalysis.Razor.Completion
     internal enum RazorCompletionItemKind
     {
         Directive,
+        DirectiveSnippet,
```

Comment:
Hmmm digging through I must have missed but why did we need to add a new completion item kind? Where did `RazorCompletionItemKind.Directive` paired with the `IsSnippet` fall short?

Reply:
I was gonna say "Because we need to show snippet icon", but as I typed this answer I realized, that this new kind is definitely not required to do it)